### DISCUSSION TITLE
Presentation mode unclickable in right sidebar

### DISCUSSION THREAD

**Discussion by Joe-Dough** [stephanrauh/ngx-extended-pdf-viewer#1806] [Q&A] [answered]
Hello,

When the presentation mode button goes in the sidebar after a screen resize, the button seems to be unclickable. 
The button is showing but nothing happens when the button is pressed.

Any ideas?

**Comment by stephanrauh** [comment, marked as answer]
Well, you've found a bug that's already been lurking in the library for ten months. The pdf.js team tidied up their toolbar, remove the presentation mode button from the main toolbar, and renamed the id of the secondary menu item. My HTML code didn't change, but the id of the principal toolbar button matched the id of Mozilla's menu item, so I never noticed.

I hope the Playwright tests I'm currently writing prevent this kind of nasty surprise in future.

Here's the commit that caused the bug:
https://github.com/mozilla/pdf.js/commit/e56c30eb59f302b450deebaab68242a84a17722e

**Reply by stephanrauh** [reply]
Please update to version 17.4.4 (stable branch only) or 18.0.0-alpha.1. They solve the bug.

Enjoy!

**Reply by Joe-Dough** [reply]
Great! Thank you for the speedy update!
